### ISSUE TITLE
SAML Auth Error "service doesn't match requested authentication method"

### ISSUE THREAD

**Issue by DCUKWS** [OpenCTI-Platform/opencti#7607]
## Prerequisites

- [ X ] I read the [Deployment and Setup](https://docs.opencti.io/latest/deployment/overview) section of the OpenCTI documentation as well as the [Troubleshooting](https://docs.opencti.io/latest/deployment/troubleshooting) page and didn't find anything relevant to my problem.
- [ X ] I went through old GitHub issues and couldn't find anything relevant
- [ X ] I googled the issue and didn't find anything relevant

## Description
i am using SAML strategy, it worked ok since using it more than a year, until recently when enabling `Windows Hello`
now i get the error `AADSTS75011: Authentication method 'X509, MultiFactor, X509Device' by which the user authenticated with the service doesn't match requested authentication method 'Password, ProtectedTransport'. Contact the OpenCTI application owner.`

## Environment

1. OS: EKS
2. OpenCTI version: LATEST
3. OpenCTI client: Frontend
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. enable windows hello
2. click the Login button on the GUI frontend
3. `AADSTS75011: Authentication method 'X509, MultiFactor, X509Device' by which the user authenticated with the service doesn't match requested authentication method 'Password, ProtectedTransport'. Contact the OpenCTI application owner.`

## Additional information
i have tried different permutations of the config, Examples below

```
     PROVIDERS__SAML_CONFIG__disableRequestedAuthnContext:                                                                    
      value: "true"                                                                                                        
    PROVIDERS__SAML_CONFIG__authnContext:                                                                                    
      value: "[http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/password, http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/x509]"
```
ref : (https://github.com/OpenCTI-Platform/opencti/issues/2429)

 ```
      PROVIDERS__SAML_CONFIG__authnContext:
        value: "https://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/unspecified"
```
or
```
      PROVIDERS__SAML_CONFIG__authnContext:
        value: "[urn:oasis:names:tc:SAML:2.0:ac:classes:unspecified]"
```
ref : (https://answers.laserfiche.com/questions/207103/Error-while-using-SAML-authentication-Authentication-method-X509-MultiFactor)

**Comment by DCUKWS** [comment]
new ref i just stumbled across : (https://medium.com/@namsoochoi/solved-error-aadsts75011-during-saml-sso-a4abf7ff3946)
so, is unspecified an option and I'm just not configuring correctly ?

**Comment by DCUKWS** [comment]
mystery solved

   \-   PROVIDERS__SAML_CONFIG__disableRequestedAuthnContext:
   \+   PROVIDERS__SAML__CONFIG__DISABLE_REQUESTED_AUTHN_CONTEXT:
        value: "true"
